ui/pages/send/send-content/send-content.component.js

```diff
@@ -124,6 +140,36 @@ export default class SendContent extends Component {
     );
   }
 
+  renderRecipientWarning() {
+    const { acknowledgeRecipientWarning } = this.props;
+    const { t } = this.context;
+    return (
+      <div className="send__warning-container">
+        <ActionableMessage
+          type="danger"
+          useIcon
+          iconFillColor="#d73a49"
+          primaryActionV2={{
+            label: t('tryAnywayOption'),
+            onClick: () => acknowledgeRecipientWarning(),
+          }}
+          message={t('sendingToTokenContractWarning', [
+            <a
+              key="contractWarningSupport"
+              target="_blank"
+              rel="noopener noreferrer"
+              className="send__warning-container__link"
+              href="https://metamask.zendesk.com/hc/en-us/articles/360020028092-What-is-the-known-contract-address-warning-"
```

Comment:
[REQ] There's a new zendesk urls constant in the constants folder we should use for this.

Reply:
done